docs/reference/query_types/parameters/common/query/offset.rst.inc

```diff
@@ -2,12 +2,13 @@
 ~~~~~~~~~~
 
 Defines the offset for search hits, used for paging the results.
+If ``null`` is used as a value, the offset won't be set.
```

Comment:
The query object still has `0` for the default offset, so while `null` won't be set on it, the resulting offset will still be `0`. That's not obvious, but it's what we should document, since that's what the user will end up with.

Not sure, but maybe normalizers can be used to convert `null` to `0`.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/187/commits/31e62e94e070fe6f8e0616ae050d6551fc4e2eb4